src/AccessToken.php

```diff
@@ -129,7 +133,7 @@ public function verify($token, array $options = [])
         }
         try {
             if ($alg == 'RS256') {
-                return $this->verifyRs256($token, $certs, $audience);
+                return $this->verifyRs256($token, $certs, $audience, $issuer);
             }
             return $this->verifyEs256($token, $certs, $audience);
```

Comment:
You need to pass it here also!

Reply:
Done in 94a110effd2a6ce370007169e17e7002febad965